docs/API_ref.md

```diff
@@ -1,87 +1,57 @@
-| labels        | map [ [string](https://pkg.go.dev/builtin#string) ] [string](https://pkg.go.dev/builtin#string)                                                                     | Labels to add to pods                                                             |
-| nodeSelector        | map [ [string](https://pkg.go.dev/builtin#string) ] [string](https://pkg.go.dev/builtin#string)                                                                     | NodeSelector defines the nodeSelector to be supplied to Velero/Restic podSpec                                                             |
-| tolerations         | [][corev1.Toleration](https://pkg.go.dev/k8s.io/api/core/v1#Toleration)                  | Tolerations defines the list of tolerations to be applied to Velero Deployment/Restic daemonset                                                             |
-| resourceAllocations | [corev1.ResourceRequirements](https://pkg.go.dev/k8s.io/api/core/v1#ResourceRequirements) | Set specific resource  `limits`  and  `requests`  for the Velero/Restic pods. For more information, go  [here] ( config/resource_req_limits.md). |
-
-### [Features](https://pkg.go.dev/github.com/openshift/oadp-operator@master/api/v1alpha1#Features)
-| Property  | Type      | Description                                                                                                                               |
-|-----------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------|
-| dataMover | [DataMover](https://pkg.go.dev/github.com/openshift/oadp-operator@master/api/v1alpha1#DataMover) | DataMover defines the various config for DPA data mover                                                             |
-
-### DataMover
-| Property       | Type | Description                                                                                                                   |
-|----------------|------|-------------------------------------------------------------------------------------------------------------------------------|
-| enable         | [bool](https://pkg.go.dev/builtin#bool) | Enable is used to specify whether you want to deploy the volume snapshot mover controller and a modified csi datamover plugin |
-| credentialName | [string](https://pkg.go.dev/builtin#string) | User supplied Restic Secret name for DataMover |                                                                               |
-| timeout        | [string](https://pkg.go.dev/builtin#string) | User supplied timeout to be used for VolumeSnapshotBackup and VolumeSnapshotRestore to complete, default value is 10m |                                                                               |
-
-See also [![Go Reference](https://pkg.go.dev/badge/github.com/openshift/oadp-operator.svg)](https://pkg.go.dev/github.com/openshift/oadp-operator) for a deeper dive.
+OADP install CRDs for the following resources
```

Comment:
Instead of giving the instructions to install the CRDs manually, Should we just add a note that the user should make sure they have OADP operator installed on their cluster or something ?

Reply:
I didn't mean to instruct them to install anything manually.. was just listing them out :D

Reply:
```suggestion
Pre-requisites: Install OADP to your cluster. before proceeding to the next steps.
```